File: src/main/c#/Lab 6 CSharp/Lab6/Publication.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Lab6
{
    public class Publication
    {
        private String title;
        private Double price;
        private int copies;

        public String Title
        {
            get { return title; }
            set { this.title = value; }
        }

        public Double Price
        {
            get { return price; }
            set { price = value; }
        }

        public int Copies
        {
            get { return copies; }
            set { copies = value; }
        }

        public String getTitle() { return title; }

        public void setTitle(String title) { this.title = title; }

        public int getCopies() { return copies; }

        public void setCopies(int copies) { this.copies = copies; }

        public Double getPrice() { return price; }

        public void setPrice(Double price) { this.price = price; }

        public Publication()
        {
            this.copies = 0;
            this.price = 0.0d;
            this.title = "No title";
        }

        override public String ToString()
        {
            return "Title: " + title +
                "\nPrice: " + "$" + price +
                "\nCopies: " + copies;
        }

        public void sellCopy()
        {
            Console.WriteLine("Sell Copy");
        }

    }
}

File: src/main/java/Lab3/Publication.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package Lab3;

/**
 *
 * @author 14034305
 */
public class Publication {
    private String title;
    private double price;
    private int copies;
    
    
    
    public Publication(){
    
}
    
    public Publication(String title, double price, int copies){
        this.title = title;
        this.price = price;
        this.copies = copies;        
    }
    
    public void sellCopy(){        
    }
    
    @Override
    public String toString(){
        String s="";
        s+= "\nTitle: " + getTitle()+
            "\nPrice: " + getPrice()+
            "\nCopies: " + getCopies();  
        return s;
    }

    /**
     * @return the title
     */
    public String getTitle() {
        return title;
    }

    /**
     * @param title the title to set
     */
    public void setTitle(String title) {
        this.title = title;
    }

    /**
     * @return the price
     */
    public double getPrice() {
        return price;
    }

    /**
     * @param price the price to set
     */
    public void setPrice(double price) {
        this.price = price;
    }

    /**
     * @return the copies
     */
    public int getCopies() {
        return copies;
    }

    /**
     * @param copies the copies to set
     */
    public void setCopies(int copies) {
        this.copies = copies;
    }
}
